contracts/FantomAuction.sol

```diff
@@ -379,7 +375,7 @@ contract FantomAuction is OwnableUpgradeable, ReentrancyGuardUpgradeable {
         // Refund the top bidder
         _refundHighestBidder(_nftAddress, _tokenId, _msgSender(), previousBid);
 
-        emit BidWithdrawn(_nftAddress, _tokenId, _msgSender(), previousBid);
+        emit TransferBid(_nftAddress, _tokenId, _msgSender(), previousBid);
```

Comment:
Same thing here. Can we keep `BidWithdrawn`?

Reply:
Yup noted